data/ka'het/ka'het missions.txt

```diff
@@ -293,3 +293,411 @@ mission "Ringworld Debris: Quarg"
+			label dontcare
+			`	"I need to prepare a few more instruments before we land," Dusk says as he stops staring at the dome and returns to his equipment. As you reach the complex, you do a few flights around it in search for a good landing site. Dusk indicates to you a spot right next to a set of small circular indents on the structure. After touching down, you and Dusk don extravehicular suits and start walking towards the cupola. He stops in front of one ring and starts looking through his equipment.`
+			choice
+				`	"What is that?"`
+			`	"Well, I obviously cannot be sure, but given its location and that on the other side there is a pressurized area, it should be an airlock. I have seen scans of the Builder stations, and they look quite similar." He takes a heavy-looking scanner, puts it on the wall of the structure and starts moving it around. "If I am right, this way should have a control station hidden inside the wall - and here it is!" he sings with satisfaction.`
+			`	Using a small instrument similar in appearance to a crowbar, Dusk slowly pulls out a section from the wall, kicking off a good amount of dust in the process. The control station is a two sided panel, full of switches and buttons with small alien markings above each of them.`
+			choice
+				`	"Can you read them?"`
+				`	"Can you open the airlock?"`
+					goto open
+				`	"Are you sure this won't put us in danger?"`
+					goto danger
+			`	"I can, yes. The first row of switches controls the lights inside the complex, the second row controls redundant pressurization systems, and there are similar such trivial operations controlled by the rest of the panel.`
+				goto open
+			label danger
+			`	"There is no way to be certain, but this is the only possible way to get in without destroying half of the complex.`
+			label open
+			`	"This," Dusk says as he presses a switch in the middle of the panel, "is what we were looking for." Soon after, the door in front of you opens, revealing a small room with another door at the opposite end. You follow Dusk inside, where he takes out the scanner again to look for another control panel. Once that is open, another switch allows him to close the door. If it weren't for the lights installed in your helmet, it would now be completely dark. Looking around, you notice that the architecture is distinctly alien, with rounded angles on every side of the room and slides going down of both doors. The ring itself is too small for you to pass through it standing upright, but the low gravity of the asteroid lets you almost fly though it without much trouble. "We now have to wait for the room to pressurize," he tells you.`
```

Comment:
```suggestion
			`	"This," Dusk says as he presses a switch in the middle of the panel, "is what we were looking for." Soon after, the door in front of you opens, revealing a small room with another door at the opposite end. You follow Dusk inside, where he takes out the scanner again to look for another control panel. Once that is open, another switch allows him to close the door. If it weren't for the lights installed in your helmet, it would now be completely dark. Looking around, you notice that the architecture is distinctly alien, with rounded angles on every side of the room and slides going down off of both doors. The ring itself is too small for you to pass through it standing upright, but the low gravity of the asteroid lets you almost fly though it without much trouble. "We now have to wait for the room to pressurize," he tells you.`
```
Not sure what the original meaning of `slides going down of both doors` was, but doesn't make sense as-is. If adding an `off` changes the meaning to something you didn't intend, we can definitely play around with it.